graphql-database-manager-test/src/main/java/com/fleetpin/graphql/database/manager/test/TestDatabaseProvider.java

```diff
@@ -45,23 +47,29 @@
 
         server = startDynamoServer(port);
         final var client = startDynamoClient(port);
+        final var streamClient = startDynamoStreamClient(port);
 
         System.setProperty("sqlite4java.library.path", "native-libs");
 
         finished = new CompletableFuture<>();
 
         final var testMethod = extensionContext.getRequiredTestMethod();
         final var organisationId = testMethod.getAnnotation(TestDatabase.class).organisationId();
+        final var withHistory = testMethod.getAnnotation(TestDatabase.class).withHistory();
```

Comment:
we could get rid of this withHistory setting on the annotation and just look to see if one of the parameters is .isAssignableFrom(HistoryProcessor.class)